libs/wingsdk/src/target-sim/schema-resources.ts

```diff
@@ -312,6 +351,10 @@ export interface DynamodbTableSchema extends BaseResourceSchema {
      * Range key for this table.
      */
     readonly rangeKey?: string;
+    /**
+     * The GSI for the table.
+     */
+    readonly globalSecondaryIndex?: DynamodbTableGlobalSecondaryIndex[];
```

Comment:
can `DynamodbTableGlobalSecondaryIndex` be imported from `libs/wingsdk/src/ex/dynamodb-table.ts`?

Reply:
I added to `libs/wingsdk/test/target-tf-aws/dynamodb-table.test.ts`.

It looks like the tests in `libs/wingsdk/test/target-sim/dynamodb-table.test.ts` will have to test exactly what I've already tested in `examples/tests/sdk_tests/dynamodb-table/global-secondary-index.test.w`, is it still relevant to add there too?

Reply:
yes, the `.w` test files run on the cloud before publishing a version, and the `.ts` tests run on each PR, keeping us covered and preventing regression issues 💪

Reply:
Thank you for the clarification 👍, I added that one too.